app/integrations/sequencescape/api/v2/tube_rack.rb

```diff
@@ -17,19 +17,22 @@ class Sequencescape::Api::V2::TubeRack < Sequencescape::Api::V2::Base
   STATE_EMPTY = 'empty'
   STATE_MIXED = 'mixed'
 
-  # This is needed in order for the URL helpers to work correctly
+  # Overrides the Rails method to return the UUID of the labware for use in URL generation.
+  #
+  # @return [String] The UUID of the labware instance.
   def to_param
+    # Currently use the uuid as our main identifier, might switch to human barcode soon
     uuid
   end
 
   #
   # Override the model used in form/URL helpers
   # to allow us to treat old and new api the same
   #
-  # @return [ActiveModel::Name] The resource behaves like a Limber::TubeRack
+  # @return [ActiveModel::Name] The resource behaves like a TubeRack
   #
   def model_name
-    ::ActiveModel::Name.new(Limber::TubeRack, false)
+    ::ActiveModel::Name.new(Sequencescape::Api::V2::TubeRack, false, 'TubeRack')
```

Comment:
For Plate and Tube, there was only the class with a shorter syntax. Here it uses a fully qualified class for TubeRack. Is this intentional?